src/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -23,87 +23,51 @@ GetCLRRuntimeHost
 ; Win32 API and other PAL functions used by the mscorlib
 CloseHandle
 CoCreateGuid
-CopyFileW
```

Comment:
What is the purpose of mscoree in .NET Core? There seems to be no traditional mscoree.dll but it is linked into some utilities.

Reply:
There is no mscoree in .NET Core. (Except that the managed .exes still have a dummy reference to it.)

Some of the methods that used to be exported from mscoree are exported from coreclr instead as .NET Core internal implementation details.

Reply:
So this file serves no purpose (except for those exe's)? Do we need the mscoree folder, ideally?

Reply:
This directory builds coreclr.dll. The directory has historical name that nobody bothered to update yet.

Reply:
Oh- missed that.